azure-pipelines.yml

```diff
@@ -0,0 +1,48 @@
+jobs:
+- job: Linux
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - task: DotNetCoreInstaller@0
+    inputs:
+      version: '2.1.403'
+  - script: |
+      export COREHOST_TRACE=0
+      export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
+      export DOTNET_CLI_TELEMETRY_OPTOUT=1
+      which dotnet
+      dotnet --info
+      ./build.sh --target "Azure-Linux" --configuration "Release"
+     
+- job: macOS
+  pool:
+    vmImage: 'xcode9-macos10.13'
+  steps:
+  - task: DotNetCoreInstaller@0
+    inputs:
+      version: '2.1.403'
+  - script: |
+      export COREHOST_TRACE=0
+      export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
+      export DOTNET_CLI_TELEMETRY_OPTOUT=1
+      which dotnet
+      dotnet --info
+      ./build.sh --target "Azure-OSX" --configuration "Release"
+
+- job: Windows
+  pool:
+    vmImage: 'vs2017-win2016'
+  steps:
+  - task: DotNetCoreInstaller@0
+    inputs:
+      version: '2.1.403'
+  - script: |
```

Comment:
We should be able to change this whole block to `powershell: build.ps1 --target "Azure-Windows" --configuration "Release"